Sources/Sentry/include/SentryLog.h

```diff
@@ -15,3 +15,14 @@ SENTRY_NO_INIT
 @end
 
 NS_ASSUME_NONNULL_END
+
+#define SENTRY_LOG_DEBUG(...)                                                                      \
+    [SentryLog logWithMessage:[NSString stringWithFormat:__VA_ARGS__] andLevel:kSentryLevelDebug]
+#define SENTRY_LOG_INFO(...)                                                                       \
+    [SentryLog logWithMessage:[NSString stringWithFormat:__VA_ARGS__] andLevel:kSentryLevelInfo]
+#define SENTRY_LOG_WARN(...)                                                                       \
+    [SentryLog logWithMessage:[NSString stringWithFormat:__VA_ARGS__] andLevel:kSentryLevelWarning]
+#define SENTRY_LOG_ERROR(...)                                                                      \
+    [SentryLog logWithMessage:[NSString stringWithFormat:__VA_ARGS__] andLevel:kSentryLevelError]
+#define SENTRY_LOG_CRITICAL(...)                                                                   \
+    [SentryLog logWithMessage:[NSString stringWithFormat:__VA_ARGS__] andLevel:kSentryLevelCritical]
```

Comment:
Excellent 👌. Can you open an extra PR for this so we can use it sooner?

Reply:
Yes, I had a plan to do so. Thanks for doing it for the typos 🙏🏻

Reply:
https://github.com/getsentry/sentry-cocoa/pull/2163